.packit.yaml

```diff
@@ -10,22 +10,22 @@ jobs:
 - job: copr_build
   enable_net: true
   owner: "@oamg"
-  project: "rhc-worker-bash"
+  project: "rhc-worker"
```

Comment:
Is it because of this that the copr build is not runned on PR? I mean because of the repo still has old name

Reply:
Yes, exactly!